app/graphql/types/base_union.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Types
+  class BaseUnion < GraphQL::Schema::Union; end
+end
```

Comment:
A union can only ever be a union, it too is a language primitive that can't be extended in a meaningful way